docs/byoh-instance-pre-requisites.md

```diff
@@ -3,6 +3,8 @@
 The following pre-requisites must be fulfilled in order to add a Windows BYOH node.
 * The Docker container runtime must be installed on the instance.
 * The instance must be on the same network as the Linux worker nodes in the cluster.
+* If the cluster platform is set, the instance being added must be part of the platform.
+  * The platform of a cluster can be determined with: `oc get infrastructure cluster --template={{.status.platform}}`
```

Comment:
@sebsoto is this needed for the release? If so I suggest we merge it into #686. cc: @mansikulkarni96